.github/workflows/release.yaml

```diff
@@ -237,7 +237,10 @@ jobs:
           image-ref: ${{ matrix.url }}/${{ secrets[matrix.repository] }}:${{ needs.prepare.outputs.version }}@${{ needs.manifest.outputs.digest }}
           format: 'cyclonedx'
           output: 'result.json'
-          skip-dirs: '/usr/share/dynatrace-operator/third_party_licenses'
+          skip-dirs:
+            - '/usr/share/dynatrace-operator/third_party_licenses'
+          skip-files:
+            - '/usr/local/bin/dynatrace-operator'
```

Comment:
according to VSCode this is incorrect, and it should be a single line and not a sequence.